### ISSUE TITLE
Members / Users section

### ISSUE THREAD

**Issue by rufuspollock** [openspending/community.openspending.org#30]
A members section connected to the main Open Knowledge community directory at http://discuss.okfn.org/users/
- [x] Badge setup - https://discuss.okfn.org/badges/116/openspending
- [ ] `/members/` page exists and has info
  - Briefly what membership means
  - How to become a member - i.e. how to get listed on https://discuss.okfn.org/badges/116/openspending
  - Say that list of members is at https://discuss.okfn.org/badges/116/openspending (until we find a way to maybe inline)
## Old
- Suggest we do a members / users page a bit like OK Labs one.
  - Naming would be "users" and hence `/users/` rather than members
- Authors on pages should be usernames (if we have page authors at all ...)
- Can also have badging (e.g. esp showing roles - e.g. you are on steering group)

Questions:
- How does this relate to users on http://openspending.org/ itself?
  - Right now they are distinct and we live with that.
  - One day we hope these could be merged. However, for now can think of this as the "wiki" users separate from the app users.

**Comment by danfowler** [comment]
I think linking to https://discuss.okfn.org/users/{{ username }} is probably best.

**Comment by rufuspollock** [comment]
@danfowler yes - it would be nice to have a `/members/` page. At the the start it could just link to the relevant "badge" page on the forum.

**Comment by danfowler** [comment]
Like here: https://discuss.okfn.org/badges/116/openspending

**Comment by rufuspollock** [comment]
Yes :-)

**Comment by danfowler** [comment]
@hoony would you be interested in implementing?

**Comment by hoony-o-1** [comment]
cool! sure. where can I get users' list? do users add themselves like [okfnlabs.org](http://okfnlabs.org)?

**Comment by danfowler** [comment]
@hoony in theory, but we don't have anything worked out.  For the time being, you could simply add a `index.md` file in /members/ and add a `redirect_to: https://discuss.okfn.org/badges/116/openspending` attribute.

**Comment by rufuspollock** [comment]
@danfowler that badge is not granted to anyone much atm ;-) - e.g. could i and others start getting it :-)

Re this page on the community site. I do think having a proper page rather than a simple redirect is useful for e.g.
- Briefly what membership means
- How to become a member - i.e. how to get listed on https://discuss.okfn.org/badges/116/openspending
- Say that list of members is at https://discuss.okfn.org/badges/116/openspending (until we find a way to maybe inline)

**Comment by hoony-o-1** [comment]
Ok, I understood.
- /members/
  - Briefly what membership means
  - How to become a member
  -  list of members is at https://discuss.okfn.org/badges/116/openspending

then, what content should be on `/members/{{username}}`? no need yet?

**Comment by danfowler** [comment]
@hoony no need yet for content per member.  

How to become a member: I would say that a group member can introduce themselves on [this thread](https://discuss.okfn.org/t/openspending-community-introductions/1145) and request to be added to the OpenSpending group.

**Comment by hoony-o-1** [comment]
got it, I'm working on it. I'll send pull request of base file for it soon.

**Comment by danfowler** [comment]
Great! :+1:

**Comment by hoony-o-1** [comment]
@danfowler @rgrp could you put contents on the list that should be on the page? each headers and contents.
1. what membership means
2. How to become a member
3. what content should I list on each member on the members of list

**Comment by danfowler** [comment]
@hoony we're starting to rebuild our OpenSpending community around this new platform, so I'm open to suggestions as to what _you_ think it means to be a member of the OpenSpending community.  What do you think?

**Comment by hoony-o-1** [comment]
In my experience to organize code for seoul community, there could be two kinds of contributors of a community
1. Project Contributors
   - contributors to upload data to OpenSending
   - contributors to sub projects using OpenSpending data such as wheredoesmymonygo(every local versions)
2. Community Contributors
   - contributors to maintain a community(website contributors or discourse manager)
   - contributors to a main level project like promotion event, hosting an event or promotion website contributors

**Comment by danfowler** [comment]
@hoony excellent!  this reminds me that this issue is directly related to our discussion of how to [/get-involved/](http://community.openspending.org/get-involved/).

**Comment by hoony-o-1** [comment]
exactly! 'if you got involved!?, then you've got a membership of OpenSpending!'
